# Don't check for the latest version as it's only updated via apk anyways.
diff --git c/internal/model/cluster_info.go i/internal/model/cluster_info.go
index 5f1653be..e42713a5 100644
--- c/internal/model/cluster_info.go
+++ i/internal/model/cluster_info.go
@@ -147,9 +147,6 @@ func (c *ClusterInfo) Refresh() {
 	v1 := NewSemVer(data.K9sVer)
 
 	var latestRev string
-	if !c.cfg.SkipLatestRevCheck {
-		latestRev = c.fetchK9sLatestRev()
-	}
 	v2 := NewSemVer(latestRev)
 
 	data.K9sVer, data.K9sLatest = v1.String(), v2.String()
